engine/source/engine.cpp

```diff
@@ -306,11 +304,6 @@ void OldEngine::Tick(Engine& engine)
         audioModule.StopEvent(eventId);
     }
 
-    _ecs->UpdateSystems(deltaTimeMS);
-    _ecs->GetSystem<PhysicsSystem>()->CleanUp();
```

Comment:
Might've forgotten to put the physicssystem cleanup in the render module's tick

Reply:
Yeah I have just noticed... do you think there is a way to not explicitly place it in the ECS module? Id rather not add a dependency on the OldEngine for this cleanup function

Reply:
I'd probably suggest moving the physics system (same for the rigidbody component) away from OldEngine since it doesn't really make since being in that place anyway now that the ECS has moved. Judging from particles and renderer modules which have their components in their own modules, I'd suggest moving the physics system and rigidbody component to the physics module.

Reply:
This would mean that the ECS will have a dependency on the physics module which I hope isn't too big of an issue

Reply:
Oh thats a nice idea... will try it.